gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ui/SnappyRegionStats.java

```diff
@@ -17,7 +17,9 @@
 
 package com.pivotal.gemfirexd.internal.engine.ui;
 
-public class SnappyRegionStats {
+import java.io.Serializable;
+
+public class SnappyRegionStats implements Serializable {
```

Comment:
This can implement GfxdDataSerializable or DataSerializable as a general practice, which will work with DataSerializer.writeObject mentioned above (latter works for java serialization too, but its always better to have DataSerializable for inbuilt classes).